### PR TITLE
Builder docs

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewsxcv

### CODE REVIEW

**Comment by frewsxcv** [comment]
tidy reports trailing whitespace (see Travis).

**Comment by GuillaumeGomez** [comment]
It seems it wasn't me. Some commits have added. I rebased.

File: src/libstd/thread/mod.rs [hunk 1]
```diff
@@ -228,6 +242,22 @@ pub struct Builder {
 impl Builder {
     /// Generates the base configuration for spawning a thread, from which
     /// configuration methods can be chained.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::thread;
+    ///
```
Comment (frewsxcv) [inline src/libstd/thread/mod.rs]: There's trailing whitespace on this line for example: 

```
+    /// 
```

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by frewsxcv** [comment]
There's still trailing whitespace. See Travis.

**Comment by GuillaumeGomez** [comment]
Updated again.

File: src/libstd/thread/mod.rs [hunk 2]
```diff
@@ -228,6 +242,22 @@ pub struct Builder {
...
+    /// let builder = thread::Builder::new();
```
Comment (frewsxcv) [inline src/libstd/thread/mod.rs]: Semicolon should be removed here

Comment (GuillaumeGomez) [inline src/libstd/thread/mod.rs]: Arf, good catch!

**Comment by GuillaumeGomez** [comment]
Updated!

**Pull request merged**